src/components/DefaultImage.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+function DefaultImage({idx, onClick, src, srcSet, sizes, height, width, alt, margin}){
+	style.margin = margin;
+	return (
+		<div style={style}> 
```

Comment:
As there is a `onClick` handler; and no use of the `href`, the `div` and `a` seems to be redundant to me. Why not optimize to returning a single `<img>` ? Images can have `onClick` handlers and `style` attributes as well.

Reply:
I'll look into this in a separate issue as I think i needed it for something.  But I do like that the <a> provides the mouse pointer style without having to add it in the CSS.  Also I think I will modify the href to accept a parameter in the case the user does actually want to link the image.  There does need to be at least one containing element because I have a background color on the outer div in case the image fails to load.  Now that the user can just pass in their own image component the need for the extra markup is probably not so important.